#533 Feature: Support Configuration Store values to use properties to inject values (#545)
boozallen/aissemble: extensions/extensions-helm/aissemble-configuration-store-chart/README.md

| configMap.name                                              | The name of the config map for quarkus config properties                                                                                                                                                                                                                     | No                | config-store-quarkus-config                                                                                                                                                                                                     |
| configMap.baseSparkProperties                               | The base Spark properties for Universal Configuration                                                                                                                                                                                                                        | No                | config-store-quarkus-config                                                                                                                                                                                                     |

> | configMap.envSparkProperties                                | The environment specific Spark properties for Universal Configuration                                                                                                                                                                                                        | No                | config-store-quarkus-config                                                                                                                                                                                                     |

Reviewer: These should be in the config map table below

Author: those actually can be deleted, forgot to cleanup my previous logic

| toggleableConfiguration.configMap.supplementalQuarkusConfig | The toggleable list of additional properties to provide to the Quarkus app                                                                                                                                                                                                   | No                | `- quarkus.http.ssl.certificate.file=/etc/webhook/cert/tls.crt`<br/>`- quarkus.http.ssl.certificate.key-file=/etc/webhook/cert/tls.key`                                                                                         |
| configStore.baseVolumeMounts                                | The ConfigStore base volume mounts for the pod                                                                                                                                                                                                                               | No                | `/etc/webhook/cert=certs`                                                                                                                                                                                                       |

## Configuration Store Properties

> The following properties are used injected in configMap to mount configuration properties file in the downstream project.

Reviewer: ```suggestion
The following properties are used to create a configMap for defining and mounting configuration properties files.
```

> The following properties are used injected in configMap to mount configuration properties file in the downstream project.

Reviewer: This content and table below should be moved to the `### Config Map Properties` section.

Author: updated!

| toggleableConfiguration.configMap.supplementalQuarkusConfig | The toggleable list of additional properties to provide to the Quarkus app                                                                                                                                                                                                   | No                | `- quarkus.http.ssl.certificate.file=/etc/webhook/cert/tls.crt`<br/>`- quarkus.http.ssl.certificate.key-file=/etc/webhook/cert/tls.key`                                                                                         |
| configStore.baseVolumeMounts                                | The ConfigStore base volume mounts for the pod                                                                                                                                                                                                                               | No                | `/etc/webhook/cert=certs`                                                                                                                                                                                                       |

> ### Persistent Volume Properties

Reviewer: Should have a header before this for `## Configuration Store Properties`

          value: <Optional URI housing property overrides>
```

### Config Map Properties

#### Defining Properties Files

> The Configuration Store support simplified way to provide configurations instead of mounting properties files.

Reviewer: ```suggestion
The Configuration Store supports a simplified way of providing configurations without mounting properties files.
```

> The user can define its configuration properties within values.yaml file as below:

Reviewer: Can moved this above `#### Defining Properties Files` and below `### Config Map Properties`

Author: hmm do you mean move whole paragraph? 

The Configuration Store support simplified way to provide configurations instead of mounting properties files.
The user can define its configuration properties within values.yaml file as below: ...
```yaml
configMapProperties: ...

Reviewer: ```
### Config Map Properties
The Configuration Store support simplified way to provide configurations instead of mounting properties files.

#### Defining Properties Files
The user can define its configuration properties within values.yaml file as below:
```

> The user can define its configuration properties within values.yaml file as below:

Reviewer: ```suggestion
The user can define its configuration properties within their values file as below:
```
